lib/osf-components/addon/components/osf-dialog/component.ts

```diff
@@ -30,6 +30,12 @@ export default class OsfDialog extends Component {
 
     @or('isOpen', 'hasTriggeredOpen') shouldBeOpen!: boolean;
 
+    @computed('renderInPlace')
+    get _renderInPlace() {
+        return this.renderInPlace || typeof document === 'undefined' ||
+            !document.getElementById('osf-dialog-wormhole');
```

Comment:
Can't you just check if `this.osfModalState.dialogWormholeTarget` is set?

Reply:
Right. I will check for `this.osfModalState.dialogWormholeTarget` instead.